### ISSUE TITLE
Broken link to the docs

### ISSUE THREAD

**Issue by WooDzu** [serverless/serverless#2072]
latest serverless:

`$ serverless deploy`
 
```
  Error --------------------------------------------------
 
     ServerlessError: ServerlessError: AWS provider credentials
     not found. You can find more info on how to set up provider
     credentials in our docs here: https://git.io/v64aN
```

https://git.io/v64aN points at:
https://github.com/serverless/serverless/blob/master/docs/guide/provider-account-setup.md#setting-the-aws-api-key--secret
should be:
https://github.com/serverless/serverless/blob/master/docs/02-providers/aws/01-setup.md

**Comment by flomotlik** [comment]
Thanks for bringing it up, fixed in #2077

**Comment by joshgarwood** [comment]
This is broken again... Are the shortlinks expiring?
